disco_aws_automation/disco_aws.py

```diff
@@ -193,10 +193,9 @@ def get_subnets(self, meta_network, hostclass):
         subnet_ips = self._get_hostclass_ip_address(hostclass)
 
         if not subnet_ips:
-            return meta_network.subnets
+            return [subnet.subnet for subnet in meta_network.subnets]
```

Comment:
A general suggestion for this topic in this file: it looks like a lot of things get variables named "subnet" in the various related functions.  Is it possible to make those variable names more context specific, so it's more clear when (for example) we're looking at a local object, an API-returned dictionary, or an actual CIDR string?